pyramid_swagger/tween.py

```diff
@@ -9,6 +9,7 @@
 
 import bravado_core
 from bravado_core.exception import SwaggerMappingError
+from bravado_core.formatter import SwaggerFormat  # noqa
```

Comment:
This is done so that the user can import SwaggerFormat from `pyramid_swagger` instead of importing `bravado-core` and importing from there.